config/database.yml

```diff
@@ -17,11 +17,21 @@
 development: &default
   adapter: postgresql
   encoding: unicode
-  database: champaign
+  database: champaign_test
```

Comment:
Yeah - I see there's some database config mix-up on the worker (it's
configed as champaign_test in the compose file).

On Tue, Aug 25, 2015 at 9:48 AM, Omar Sahyoun <notifications@github.com>
wrote:

> In config/database.yml
> <https://github.com/SumOfUs/Champaign/pull/85#discussion_r37889066>:
>
> > @@ -17,11 +17,21 @@
> >  development: &default
> >    adapter: postgresql
> >    encoding: unicode
> > -  database: champaign
> > +  database: champaign_test
>
> Looks good on champaign-ai-processor:
> https://github.com/SumOfUs/champaign-ak-processor/blob/development/config/database.yml#L4
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SumOfUs/Champaign/pull/85/files#r37889066>.
>